client/curriculum/html/AboutHtml.tsx

```diff
@@ -0,0 +1,335 @@
+            {' '}
+            <em>HTML</em>
+            . Asa cum o casa incepe cu fundatia, constructia unei aplicatii web incepe
+            cu HTML.
+          </p>
+          <p>
+            Dupa ce structura de baza este gata, urmeaza sa definim culorile folosite,
+            plasamentul geamurilor si a usilor si alte lucruri ce tin de design.
+            Pentru un programator front-end, acesta este pasul in care se introduce
+            {' '}
+            <em>CSS</em>
+            -ul.
+          </p>
+          <p>
+            La final, trebuie ca geamurile, usile si becurile puse la pasul anterior sa
+            functioneze corespunzator si sa putem interactiona cu ele. Asa cum probabil
+            ati ghicit, in cazul unei aplicatii web, aceste operatii ii revin
+            {' '}
+            <em>Javascript</em>
+            -ului.
+          </p>
+          <p>
+            Noi ne vom concentra astazi pe primul limbaj dintre cele trei enumerate
+            mai sus, si anume HTML.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="intro">
+            Ce este HTML?
+          </LessonHeading>
+          <p>
+            <strong>HTML (Hyper Text Markup Language)</strong>
+            {' '}
+            este limbajul standard de markup folosit pentru a defini
+            structura de baza a unei pagini web. Nu este un limbaj de
+            programare propriu-zis, ci doar un limbaj descriptiv, bazat pe
+            {' '}
+            <em>XML</em>
+            .
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="elemente">
+            Elemente
+          </LessonHeading>
+          <p>
+            Orice fisier HTML este construit din elemente, numite si
+            <strong>tags</strong>
+            . Ele sunt formate din perechi de forma
+            {' '}
+            <strong className="formatted">{'<element> </element>'}</strong>
+            . Primul se numeste
+            {' '}
+            <strong>opening tag</strong>
+            , iar al doilea
+            {' '}
+            <strong>closing tag</strong>
+            . In interiorul acestei perechi, putem avea alte elemente sau
+            continut de tip text, precum in exemplul de mai jos.
+          </p>
+
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<section>
+  <h1>Section title</h1>
+  <p>Section description</p>
+</section>
+`}
+          />
+          <p>
+            Concret, elementul
+            {' '}
+            <strong className="formatted">{'<section>'}</strong>
+            {' '}
+            are drept continut alte doua elemente (
+            <strong className="formatted">{'<h1>'}</strong>
+            {' si '}
+            <strong className="formatted">{'<p>'}</strong>
+            ), in timp ce aceste doua elemente au in interior continut
+            de tip text.
+          </p>
+          <LessonTip>
```

Comment:
Self-closing elements feel pretty important to me so I wouldn't just use a `<LessonTip>` for them.

I would add a separate short paragraph and the code.